kibble/api/tv.go

```diff
@@ -74,6 +74,11 @@ func AppendAllTVShows(cfg *models.Config, site *models.Site, itemIndex models.It
 func AppendTVSeasons(cfg *models.Config, site *models.Site, slugs []string, itemIndex models.ItemIndex) error {
 
 	sort.Strings(slugs)
+
+	if len(slugs) > 300 {
+		slugs = slugs[:300]
+	}
+
```

Comment:
This will slice the first 300, but do we need to do this in a loop to deal with the rest?

Reply:
its does the loop where its called https://github.com/shift72/kibble/blob/master/kibble/api/site.go#L113 which is the same as how it currently does the films